File: class and object and method.py
class Car:

    name = ''
    model = ''

    def __init__(self,name,model):
        self.name = name
        self.model = model
        print('Welcome to your new Car. You can use startengine method() ,stopengine(),direction(),CheckTopspeed() , carinfo() ')

    def startengine(self):

        print("ENGINE STARTED....")

    def stopengine(self):
        print("Enginee turnedd offf")

    def checktopSpeed(self):
        return 240
    
    def direction(self):

        a = input('Please Enter where you want to go : ')

        print(f'It will take 30 mins to reach {a}')
    def carinfo(self):
        print(f'Name : {self.name} {self.model}')


myCar = Car('Audi' , 'A4')

myCar.carinfo()
        

File: class and object and method.cpp
#include <bits/stdc++.h>
using namespace std;

class Car {
public:
    string name;
    string model;


    Car(string name, string model) {
        this->name = name;
        this->model = model;
        cout << "Welcome to your new Car. You can use startEngine(), stopEngine(), direction(), checkTopSpeed(), carInfo()" << endl;
    }

    void startEngine() {
        cout << "ENGINE STARTED...." << endl;
    }

    void stopEngine() {
        cout << "Engine turned off" << endl;
    }

    int checkTopSpeed() {
        return 240;
    }

    void direction() {
        string destination;
        cout << "Please Enter where you want to go: ";
        cin >> destination;
        cout << "It will take 30 mins to reach " << destination << endl;
    }

    void carInfo() {
        cout << "Name: " << name << " " << model << endl;
    }
};

int main() {
    Car myCar("Bmw", "m3");
    myCar.carInfo();
    int a = myCar.checkTopSpeed();
    cout<<a;
    return 0;
}
